### PR TITLE
nightmare vision goggles give you night vision

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
what the title says, it gives you fullbright

## Why It's Good For The Game
it'd be really funny to see people use it in dark areas for the night vision while trying to work with the full red/black screen

## Changelog
:cl:
balance: nightmare vision goggles give you night vision
/:cl:

### CODE REVIEW

**Comment by ChungusGamer666** [comment]
next pr: nightmare vision goggles give you mare vision

File: code/modules/clothing/glasses/_glasses.dm [hunk 1]
```diff
@@ -679,6 +679,7 @@
 	inhand_icon_state = "glasses"
 	glass_colour_type = /datum/client_colour/glass_colour/nightmare
 	forced_glass_color = TRUE
+	lighting_cutoff = LIGHTING_CUTOFF_FULLBRIGHT
```
Comment (LemonInTheDark) [inline code/modules/clothing/glasses/_glasses.dm:682]: are you sure you want fullbright? we use it almost nowhere cause it ends up looking really flat. 
do you have before/after photos?

Comment (LemonInTheDark) [reply code/modules/clothing/glasses/_glasses.dm:682]: nvm

Comment (ChungusGamer666) [reply code/modules/clothing/glasses/_glasses.dm:682]: well it should be a nightmare

**Pull request merged**